torch/csrc/jit/passes/normalize_ops.cpp

```diff
@@ -90,6 +90,7 @@ const std::unordered_map<Symbol, Symbol>& getOperatorAliasMap() {
       {aten::special_erf, aten::erf},
       {aten::special_erfc, aten::erfc},
       {aten::special_erfinv, aten::erfinv},
+      {aten::special_expit, aten::sigmoid},
```

Comment:
What about special.logit?

Reply:
For logit, I decided to stick with the signature of `scipy.special.logit` which doesn't have `eps` argument.
Thus `special_logit` is not actually an alias to `logit` as the signature is different.
Let me know what you feel about the same.

Reference: 
https://pytorch.org/docs/stable/generated/torch.logit.html
https://docs.scipy.org/doc/scipy/reference/generated/scipy.special.logit.html

Reply:
I would just add the eps. It's pretty common when implementing a NumPy (or SciPy) function that it's missing some functionality from the current Torch version, and the only way we can really tell people to use the new thing is if we extend it. Since this functionality is a superset of the current special.logit functionality there's no compatibility concern with adding it.